pystac/validation/__init__.py

```diff
@@ -19,11 +19,15 @@
 from pystac.validation.stac_validator import JsonSchemaSTACValidator, STACValidator
 
 
-def validate(stac_object: STACObject) -> list[Any]:
+def validate(
+    stac_object: STACObject,
+    validator: STACValidator | None = None,
+) -> list[Any]:
     """Validates a :class:`~pystac.STACObject`.
 
     Args:
         stac_object : The stac object to validate.
+        validator: The validator to employ.
```

Comment:
Expand this description. It should be clear that the validator will be used instead of rather than in addition to the default.